### ISSUE TITLE
Hover outside the document & IE explorer issue

### ISSUE THREAD

**Issue by igor-tvardyi** [handsontable/handsontable#4362]
Hello!

Version of library 1.11.0

First thing - just hover on vertical scroll of document and you will see 2 errors in your console.
TypeError: Cannot read property 'contains' of undefined

It happens when element is document
Your code
`  _hasClass = function _hasClass(element, className) {
    if (className === '') {
      return false;
    }

    **return element.classList.contains(className);**
  };`

And the second - IE does not scroll table at all, because it forms vertical table scrollbar
![image](https://user-images.githubusercontent.com/19648531/27790548-96c55926-5ff9-11e7-830f-292e04483122.png)

I use html
`<div class="table-responsive" style="height: auto;" id="tableWrapper">
                    <div id="myTable" class="hot handsontable htRowHeaders htColumnHeaders"></div>
                </div>`

Moreover, when I use css rule contains "height: auto;" table is invisible, So I must use style="height: auto;"

`#tableWrapper {
    height: auto;
}`
It does not work. Table is invisible in this case.

**Comment by AMBudnik** [comment]
Hi @d-aven 

sorry for a delay.

Are you able to replicate the issue using this demo https://docs.handsontable.com/pro/1.11.0/demo-scrolling.html ?
I've tested IE11 on Windows 10 with the Handsontable 1.11.0 and 1.12.0 and couldn't reproduce the issue. Is there any possibility for you to record it?

**Comment by d-aven** [comment]
Hello Aleksandra,

Just modify this demo:
1) set style="height:auto; overflow:hidden" for table container or parent container (does not matter)
2) do not specify height in js

And you will have this:
http://jsfiddle.net/c6hvjzgL/
IE11 scrolling issue when hover on the table (not only header)

Thank you!

**Comment by AMBudnik** [comment]
Thanks for sharing @d-aven 

I've hovered over cells and it was OK until I got the cursor on the header, that scrolling got stuck.

**Comment by d-aven** [comment]
I have recorded video. I do not touch the header.
http://dropmefiles.com/7nDZN

**Comment by AMBudnik** [comment]
Aah, ok - sorry I misunderstood the scenario. I got the same effect as in your video on Edge as well.

**Comment by AMBudnik** [comment]
Hi @igor-tvardyi thank you for your continuous prompt replies.

As [support for the Internet Explorer browser is to be deprecated in version 8.0.0](https://github.com/handsontable/handsontable/issues/7026) we decided to close all the issues that are only replicable on that browser. All IE-related workarounds implemented into Handsontable are still a part of the code, and we do not plan to remove them.

I have tested the following issue on my Windows 10 and Edge
<img width="153" alt="edge_ver" src="https://user-images.githubusercontent.com/10757813/85010440-4e7ccd00-b160-11ea-9b46-6207cd903525.png">
and I have not experienced the same issue as before. The version of Handsontable that I tested is 8.0.0-beta2-rev15. That is why I close this issue as **solved**.
